src/database/database.h

```diff
@@ -29,6 +33,8 @@ class DatabaseManager {
     void close_database();
     std::string read_schema_from_file(const std::string& file_name);
     static int callback(void* data, int argc, char** argv, char** /* Not used */);
+    static int callback_get_id(void* not_used, int argc, char** argv, char** colName);
+    static int callback_load_tracks(void* file_arr, int argc, char** argv, char** colName);

```

Comment:
Ideally all of these new member functions should be tested within the database tests